coursemapper-kg/concept-map/src/pipelines/concept_map.py

```
@@ -62,13 +62,13 @@ def create_concept_map(self, graph: Graph, push_log_message: Callable, material_
 
         file_embedding = self.embedding_service.encode(' . '.join(pages_df['page']))
 
-        material_node = Node(material_id, material_name, '', 'material', 1, '', '\n\n'.join(pdf_text), True, file_embedding, [], Config.EMBEDDING_MODEL)
+        material_node = Node(material_id, material_name, '', 'material', 1, '', '\n\n'.join(pdf_text), True, False, False, file_embedding, [], Config.EMBEDDING_MODEL)
```

Ruff (0.8.2)

Boolean positional value in function call

(FBT003)